### DISCUSSION TITLE
TIC-80 unresponsive in fullscreen (steam Deck)

### DISCUSSION THREAD

**Discussion by Obspogon** [nesbox/TIC-80#2263] [Q&A] [answered]
I set TIC80 (itch version linux arm) to fullscreen and I cannot interact with it in any way. the window dosen't show up in game mode. launching in desktop mote shows the window, but all input is redirected to the desktop and I can't close without restarting the deck. is there any way i can delete the config on linux? or force start in windowed mode?

**Comment by Obspogon** [comment, marked as answer]
deleted `~/. local/share/com.nesbox.tic80` and reset settings

**Comment by kmturley** [comment]
FYI Steam Deck uses an x86-64 architecture not ARM
